carmajava/rsumetering/src/main/java/gov/dot/fhwa/saxton/carma/rsumetering/HoldingState.java

```diff
@@ -29,6 +29,13 @@
  * State which holds a vehicle at the ramp metering location until it is time to release it
  */
 public class HoldingState extends RSUMeteringStateBase {
+  protected static final double MIN_METER_DIST = 0.0; //meters
+  protected static final double MAX_METER_DIST = 1000.0; //meters
+  protected static final double MIN_MERGE_DIST = 5.0; //meters
```

Comment:
same as above this needs to be at least negative the length of the merge area.

Reply:
Done.